src/screenipy.py

```diff
@@ -141,102 +202,58 @@ def main(testing=False):
-                print(colorText.BOLD + colorText.FAIL + "\n[+] Script terminated by the user." + colorText.END)
-                break
-            except Fetcher.StockDataEmptyException:
-                pass
-            except Exception as e:
-                # print(colorText.FAIL + ("\n[+] Exception Occured while Screening %s! Skipping this stock.." % stock) + colorText.END)
-                if testing:
-                    raise e
+        print(colorText.BOLD + colorText.WARN +
+              "[+] Starting Stock Screening.. Press Ctrl+C to stop!\n")
+
+        items = [(executeOption, reversalOption, daysForLowestVolume, minRSI, maxRSI, respBullBear, insideBarToLookback, stock)
+                 for stock in Fetcher.listStockCodes]
+        results = pool.starmap(screenStocks, items)
+        results = list(filter(None, results))
+        for x, y in results:
+            screenResults = screenResults.append(x, ignore_index=True)
+            saveResults = saveResults.append(y, ignore_index=True)
+
         screenResults.sort_values(by=['Stock'], ascending=True, inplace=True)
         saveResults.sort_values(by=['Stock'], ascending=True, inplace=True)
         screenResults.rename(
             columns={
-                'Trend':f'Trend ({ConfigManager.daysToLookback}Days)',
-                'Breaking-Out':'Breakout-Levels'
-                },
+                'Trend': f'Trend ({ConfigManager.daysToLookback}Days)',
+                'Breaking-Out': 'Breakout-Levels'
+            },
             inplace=True
         )
         saveResults.rename(
             columns={
-                'Trend':f'Trend ({ConfigManager.daysToLookback}Days)',
-                'Breaking-Out':'Breakout-Levels'
-                },
+                'Trend': f'Trend ({ConfigManager.daysToLookback}Days)',
+                'Breaking-Out': 'Breakout-Levels'
+            },
             inplace=True
         )
         print(tabulate(screenResults, headers='keys', tablefmt='psql'))
         Utility.tools.setLastScreenedResults(screenResults)
         Utility.tools.promptSaveResults(saveResults)
-        print(colorText.BOLD + colorText.WARN + "[+] Note: Trend calculation is based on number of days recent to screen as per your configuration." + colorText.END)
-        print(colorText.BOLD + colorText.GREEN + "[+] Screening Completed! Happy Trading! :)" + colorText.END)
+        print(colorText.BOLD + colorText.WARN +
+              "[+] Note: Trend calculation is based on number of days recent to screen as per your configuration." + colorText.END)
+        print(colorText.BOLD + colorText.GREEN +
+              "[+] Screening Completed! Happy Trading! :)" + colorText.END)
         input('')
         main()
 
+
 if __name__ == "__main__":
     Utility.tools.clearScreen()
     OTAUpdater.checkForUpdate(proxyServer, VERSION)
-    main()
\ No newline at end of file
+    pool = multiprocessing.Pool(processes=multiprocessing.cpu_count())
```

Comment:
```suggestion
```
This line to be moved inside the main function to avoid `pytest: NameError`

Reply:
ok, I will do that.